.github/workflows/ci.yaml

```diff
@@ -30,85 +30,69 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-18.04]
+        catkin-build: [catkin, standalone]
         build-type: [RelWithDebInfo, Debug]
         mc-rtc-version: [head, stable]
         motion-type: [PushCartWaypoint]
         exclude:
           - build-type: Debug
             mc-rtc-version: stable
+          # Some packages have not been released onto the stable mirror yet
+          - catkin-build: standalone
+            mc-rtc-version: stable
         include:
           - os: ubuntu-20.04
             build-type: RelWithDebInfo
             mc-rtc-version: head
+            catkin-build: catkin
+            motion-type: PushCartVelMode
+          - os: ubuntu-20.04
```

Comment:
This new `include` is not needed. Instead, delete the `catkin-build: catkin` line from the original `include` above. (If I understand the syntax correctly)

Reply:
I had this in the first iteration and it only generates one configuration without any `catkin-build` defined (see https://github.com/gergondet/LocomanipController/actions/runs/4301782536/jobs/7499440259 and https://github.com/gergondet/LocomanipController/actions/runs/4301782536/workflow).

My understanding of the [doc](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) is that `include` will create new build configuration if it overwrites an existing key in the configuration and this does not include other values in the matrix (the `- fruit: banana` example in the docuentation)

Reply:
I see, thanks for the information. :ok_hand: